internal/conf/conf.go

```diff
@@ -293,6 +293,8 @@ type Conf struct {
 	RecordSegmentDuration *StringDuration `json:"recordSegmentDuration,omitempty"` // deprecated
 	RecordDeleteAfter     *StringDuration `json:"recordDeleteAfter,omitempty"`     // deprecated
 
+	RecordTimestampCSV *bool `json:"recordTimestampCSV,omitempty"` // deprecated
```

Comment:
it's not deprecated.

Reply:
I believe the actual API is, the parameters above are all deprecated.

Reply:
Do you mean the Reacord is deprecated? So ya, but we need to maintain them QQ.

Reply:
Yes, the standalone API for recording has been deprecated since 12 months ago.
We can still setup recording for each camera through the Path API.